trunk/SUAVE/Components/Wings/Control_Surface.py

```diff
@@ -20,6 +20,8 @@ def __defaults__(self):
         self.tag                   = 'control_surface'
         self.span                  = 0.0
         self.span_fraction         = 0.0
+        self.chord_fraction        = 0  # Added by Matthew 
+        self.gain                  = 0  # Added by Matthew
```

Comment:
I also don't think gain is a very clear name. A default value like downward = True might be more appropriate. It could be converted to AVL naming convention in the AVL specific files.